build_tools/bazel/glob_lit_test.bzl

```diff
@@ -0,0 +1,120 @@
+# This file is borrowed verbatim from the TensorFlow project
+# (https://github.com/tensorflow/tensorflow) and inherits the same license
+# agreement.
```

Comment:
I don't think this works - at least, this is the guidance that was provided to me in a similar situation by Google's OSS folks. Let's add the standard license headers.